README.md

```diff
@@ -345,6 +345,29 @@ You can specify multiple labels (that option will return entities which have bot
 pods = client.get_pods(label_selector: 'name=redis-master,app=redis')
 ```
 
+Get all entities of a specific type in chunks:<br>
+
+```ruby
+continue = nil
+options = {
+  limit: 1_000
+}
+
+begin
+  options[:continue] = continue unless continue.nil?
```

Comment:
I propose documenting that user may always pass `continue: nil` with same result as omitting it.
Then user could do:
```
continue = nil
begin
  entities = client.get_pods(limit: 1_000, continue: continue)

  continue = entities.continue
end until entities.last?
```

P.S. googling this [`begin ... end until ...` construct](https://rosettacode.org/wiki/Loops/Do-while#Ruby), turns out Matz [regrets it and considers removing it](http://blade.nagaokaut.ac.jp/cgi-bin/scat.rb/ruby/ruby-core/6745).  So let's write this as `loop` with `break if entities.last?` inside.

Reply:
:+1: it looks like kubernetes does handle being passed a nil continue option on the first get so yeah this is much cleaner

Reply:
Done

Reply:
kubernetes doesn't see the nil, it's dropped by
```
SEARCH_ARGUMENTS.each { |k, v| params[k] = options[v] if options[v] }
```